Examples/Swift/ViewController.swift

```diff
@@ -45,220 +60,203 @@ class ViewController: UIViewController, MGLMapViewDelegate, NavigationViewContro
-        let vertical = UITraitCollection(verticalSizeClass: .compact)
-        let traitCollection = UITraitCollection(traitsFrom: [horizontal, vertical])
-        let style = Style(traitCollection: traitCollection)
-        
-        // General styling
-        style.tintColor = #colorLiteral(red: 0.2974345386, green: 0.4338284135, blue: 0.9865127206, alpha: 1)
-        style.primaryTextColor = .black
-        style.secondaryTextColor = .gray
-        style.buttonTextColor = .black
-        
-        // Maneuver view (Page view)
-        style.maneuverViewBackgroundColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
-        style.maneuverViewHeight = 70
-        
-        // Table view (Drawer)
-        style.headerBackgroundColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
-        style.primaryTextColor = .black
-        style.secondaryTextColor = .gray
-        style.cellTitleLabelTextColor = .black
-        style.cellSubtitleLabelTextColor = .gray
-        style.cellTitleLabelFont = .preferredFont(forTextStyle: .headline)
-        style.cellSubtitleLabelFont = .preferredFont(forTextStyle: .footnote)
-        
-        return style
-    }
-    
-    func styleForiPad() -> Style {
-        let style = Style(traitCollection: UITraitCollection(userInterfaceIdiom: .pad))
-        style.maneuverViewHeight = 100
-        return style
-    }
-    
-    func styleForCarPlay() -> Style {
-        let style = Style(traitCollection: UITraitCollection(userInterfaceIdiom: .carPlay))
-        style.maneuverViewHeight = 40
-        return style
+        style.cellTitleLabelTextColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)
+        style.cellSubtitleLabelTextColor = #colorLiteral(red: 0.2549019754, green: 0.2745098174, blue: 0.3019607961, alpha: 1)
+        style.cellTitleLabelFont = UIFont(name: "Georgia-Bold", size: 17)
+        style.cellSubtitleLabelFont = UIFont(name: "Georgia", size: 15)
+            
+        style.apply()
+            
+        // If you'd like to use AWS Polly, provide your IdentityPoolId below.
+        // `identityPoolId` is a required value for using AWS Polly voice instead of iOS's built in AVSpeechSynthesizer.
+        // You can get a token here: http://docs.aws.amazon.com/mobile/sdkforios/developerguide/cognito-auth-aws-identity-for-ios.html
+        //navigationViewController.voiceController?.identityPoolId = "<#Your AWS IdentityPoolId. Remove Argument if you do not want to use AWS Polly#>"
+            
+        present(navigationViewController, animated: true, completion: nil)
     }
     
-    /// Delegate method for changing the route line style
-    func navigationMapView(_ mapView: NavigationMapView, routeStyleLayerWithIdentifier identifier: String, source: MGLSource) -> MGLStyleLayer? {
-        let lineCasing = MGLLineStyleLayer(identifier: identifier, source: source)
+    // MARK: - Navigation with multiple waypoints
+
+    func startMultipleWaypoints() {
```

Comment:
Since waypoints is coming through the pipeline in https://github.com/mapbox/mapbox-navigation-ios/pull/270, do you think we should hold off on showing developers this method for doing multi-leg routes? It's slightly an antipattern.